addon/initializers/export-application-global.ts

```diff
@@ -0,0 +1,42 @@
+import Application from '@ember/application'
+import { classify } from '@ember/string'
+
+export function initialize(application: Application): void {
+  const config:any = application.resolveRegistration('config:environment')
```

Comment:
Should probably be ` const env = application.resolveRegistration('config:environment') as { embedded?: GivenConfig }` (see https://github.com/DazzlingFugu/ember-cli-embedded/blob/master/addon/initializers/embedded.ts#L100)

I guess `GivenConfig` type (https://github.com/DazzlingFugu/ember-cli-embedded/blob/master/addon/initializers/embedded.ts#L21) will probably need to be moved in `/types`

Reply:
updated